### PR TITLE
chore: add Angle::inRadians, Angle::inArcminutes and Angle::inArcseconds python binding overload

### PR DESCRIPTION
**Pull request by AlexandreLoftOrbital**

### CODE REVIEW

File: bindings/python/src/OpenSpaceToolkitMathematicsPy/Geometry/Angle.cpp [hunk 1]
```diff
@@ -92,6 +92,13 @@ inline void OpenSpaceToolkitMathematicsPy_Geometry_Angle(pybind11::module& aModu
                 return anAngle.inRadians();
             }
         )
+        .def(
+            "in_radians",
+            +[](const Angle& anAngle, const Real& aLowerBound, const Real& anUpperBound) -> Real
+            {
+                return anAngle.inRadians(aLowerBound, anUpperBound);
+            }
+        )
```
Comment (lucas-bremond) [inline bindings/python/src/OpenSpaceToolkitMathematicsPy/Geometry/Angle.cpp:101]:
```suggestion
        .def(
            "in_radians",
            +[](const Angle& anAngle, const Real& aLowerBound, const Real& anUpperBound) -> Real
            {
                return anAngle.inRadians(aLowerBound, anUpperBound);
            },
            arg("lower_bound"),
            arg("upper_bound")
        )
```

File: bindings/python/test/geometry/test_angle.py [hunk 2]
```diff
@@ -291,3 +291,16 @@ def test_angle_in_degrees_reduction():
 
     angle: Angle = Angle(180.0, Unit.Degree)
     assert angle.in_degrees(-180.0, 180.0) == -180.0
+
+def test_angle_in_radians_reduction():
```
[the remark this answers is not in the document]
Comment (AlexandreLoftOrbital) [reply bindings/python/test/geometry/test_angle.py]: done

File: bindings/python/src/OpenSpaceToolkitMathematicsPy/Geometry/Angle.cpp (hunk 1 above)
Comment (AlexandreLoftOrbital) [reply bindings/python/src/OpenSpaceToolkitMathematicsPy/Geometry/Angle.cpp]: done

File: bindings/python/src/OpenSpaceToolkitMathematicsPy/Geometry/Angle.cpp [hunk 3]
```diff
@@ -92,6 +92,15 @@ inline void OpenSpaceToolkitMathematicsPy_Geometry_Angle(pybind11::module& aModu
                 return anAngle.inRadians();
             }
         )
+        .def(
+            "in_radians",
+            +[](const Angle& anAngle, const Real& aLowerBound, const Real& anUpperBound) -> Real
+            {
+                return anAngle.inRadians(aLowerBound, anUpperBound);
+            },
```
Comment (lucas-bremond) [inline bindings/python/src/OpenSpaceToolkitMathematicsPy/Geometry/Angle.cpp:100]: Did you try this?

```suggestion
            overload_cast<const Angle& anAngle, const Real& aLowerBound, const Real& anUpperBound>(&Angle::inRadians, const_),
```

**Review by lucas-bremond — APPROVED** [review]

**Pull request merged**